gradle.properties

```diff
@@ -50,4 +50,4 @@ text_encoding_version=0.7.0
 node_fetch_version=2.3.0
 ws_version=6.2.1
 
-org.gradle.parallel=true
+#org.gradle.parallel=true
```

Comment:
Disabling parallel may seriously slowdown the build. Is it required for train? Why?